sysl2/sysl/seqs.go

```diff
@@ -147,34 +135,36 @@ func DoGenerateSequenceDiagrams(stdout, stderr io.Writer, flags *flag.FlagSet, a
 			log.Errorln(err)
 		}
 	}()
-	var endpoints_flag, apps_flag, blackboxes_flag arrayFlags
-	root := flags.String("root", ".", "sysl root directory for input model file (default: .)")
-	endpoint_format := flags.String("endpoint_format", "%(epname)", "Specify the format string for sequence diagram endpoints. "+
-		"May include %%(epname), %%(eplongname) and %%(@foo) for attribute foo(default: %(epname))")
-	app_format := flags.String("app_format", "%(appname)", "Specify the format string for sequence diagram participants. "+
-		"May include %%(appname) and %%(@foo) for attribute foo(default: %(appname))")
-	title := flags.String("title", "", "diagram title")
-	plantuml := flags.String("plantuml", "", strings.Join([]string{"base url of plantuml server",
+	sd := kingpin.New("sd", "Generate sequence diagram")
+	root := sd.Flag("root", "sysl root directory for input model file (default: .)").Default(".").String()
+	endpoint_format := sd.Flag("endpoint_format", "Specify the format string for sequence diagram endpoints. "+
+		"May include %%(epname), %%(eplongname) and %%(@foo) for attribute foo(default: %(epname))").Default("%(epname)").String()
```

Comment:
%%(epname) is incorrect.
Only need 1 `%` character.

Reply:
Firstly I copy this description from python and I thought it's correct.
Delete 1 %.